instructure/instructure-ui
# fix(ui-calendar): fix duplicate dates for certain timezones

INSTUI-4220

more info on this: https://github.com/moment/moment/issues/4743

test plan:

- open the calendar example, set the timezone to `America/Santiago` and check 2024 september -> there should be no duplicate dates
- do the same for DateTimeInput
- check out some other timezones around the switch to or from daylight savings
